includes/class-sensei.php

```diff
@@ -377,6 +386,20 @@ private function __construct( $main_plugin_file_name, $args ) {
 		$alloptions            = wp_load_alloptions();
 		$this->install_version = $alloptions['sensei-install-version'] ?? null;
 
+		// Init the clock.
```

Comment:
Should this comment be here?

Reply:
That's the matter of consistency :)
Other code blocks also have similar comments whenever they are needed or not :)